src/main/java/org/fogbowcloud/saps/engine/core/dispatcher/utils/ProcessedImagesEmailBuilder.java

```diff
@@ -204,7 +204,7 @@ private void prepareTaskFolder(JSONObject result, SapsImage task, String folder)
 		String objectStorePath = properties.getProperty(SapsPropertiesConstants.SWIFT_OBJECT_STORE_PATH);
 		String objectStoreContainer = properties.getProperty(SapsPropertiesConstants.SWIFT_OBJECT_STORE_CONTAINER);
 		String objectStoreKey = properties.getProperty(SapsPropertiesConstants.SWIFT_OBJECT_STORE_KEY);
-		String swiftPrefixFolder = properties.getProperty(SapsPropertiesConstants.SWIFT_FOLDER_PREFIX);
+		String swiftPrefixFolder = properties.getProperty(SapsPropertiesConstants.PERMANENT_STORAGE_TASKS_FOLDER);
```

Comment:
rename to DIR instead of FOLDER (i believe we talked about it a previous review, folder is very desktop-like

Reply:
Replaced to FOLDER